### DISCUSSION TITLE
Make Cinnamenu the default menu launcher

### DISCUSSION THREAD

**Discussion by eugenialoli** [linuxmint/discussions#469] [Ideas]
I am using Cinnamenu (currently the 2nd most popular extension in your directory) and I think it should be the default launcher, configured with large icons, and fewer menu options, like this: https://files.mastodon.social/media_attachments/files/112/253/256/932/509/491/original/6183bb790dcb8365.jpg

It looks way more modern than the default menu launcher on Mint, when configured to be lean and mean. And with the thousands that will come to Mint in the coming months (I already see the trend everywhere), I think it's important to have a somewhat modern look, while not going too far away from the original UI idea. Cinnamenu fits that bill.

**Comment by Qronikarz** [comment]
It seems I accidentally recreated the Cinnamenu in my discussion:
https://github.com/orgs/linuxmint/discussions/411
Cinnamenu is good, I tested it for few days and you can configure it to behave and look exactly like the current menu, but it lacks one feature. When right clicking the menu entries it doesn't show their options. In old menu when right clicking the firefox there were few options like open new window, or open new private window. In Cinnamenu there's only those:
![menu](https://github.com/linuxmint/discussions/assets/84787215/f88e026e-686b-41fe-a935-bacf89c5c59b)

**Comment by eugenialoli** [comment]
A small price to pay if it's adopted with the large icons to look more modern and appealing. It currently looks like win7 era, and not in a good way.

**Comment by mmcdonald3009** [comment]
Have a look at this github on how to use gschema override to set default applets ( remove menu andd cinnamenu )....
https://github.com/mmcdonald3009/cinnamondesktop_cinnamenu_default

**Comment by nomad3wolf** [comment]
+1
Cinnamenu looks better than the default Menu.
This is not a critical issue to me though, because it's a simple config.

**Reply by Heavygrass** [reply]
It's a simple config for those who know how to do that.
It took me half an hour to realize how to properly setup everything.
For some people it may take much longer.

**Comment by Heavygrass** [comment]
+1
Indeed Cinnamenu is so much nicer and easier to use out of the box.
I can finally easily distinguish apps by their icons!:)
There's lots of people who choose things by its 1st appearance and Cinnamenu can bring Cinnamon to a new level from this point.
